### PR TITLE
Set Integration name scoop installs

### PR DESCRIPTION
**Pull request by maxjeffos**
- [X] Ready for review
- [X] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
Adds the integration name `SCOOP` when the running CLI instances is installed with Scoop (Windows package manager).

#### What are the relevant tickets?
HAMMER-136

### CODE REVIEW

**Review by gitphill — APPROVED** [review]

File: src/lib/analytics-sources.ts [hunk 1]
```diff
@@ -62,8 +67,50 @@ export const getIntegrationName = (args: Array<any>): string => {
...
+      const url = manifestJson.url;
+      if (
+        url.startsWith('https://github.com/snyk/snyk') &&
+        url.endsWith('snyk-win.exe')
+      ) {
+        return true;
+      }
+    }
+  } catch (error) {
+    debug('analytics-sources', error);
```
Comment (JackuB) [inline src/lib/analytics-sources.ts:113]: Would make this debug more descriptive, seeing how rare it would be

**Review by JackuB — APPROVED** [review]
Seems like you will now also have to create that fake manifest file in test to capture it

File: src/lib/analytics-sources.ts (hunk 1 above)
Comment (maxjeffos) [reply src/lib/analytics-sources.ts]: solid idea

**Comment by JackuB** [comment]
Reopening to trigger CLA check

**Pull request merged**